### PR TITLE
LL-8031 - Rebranding - Notification center

### PR DESCRIPTION
**Pull request by nparigi-ledger**
Merge https://github.com/LedgerHQ/ui/pull/103 before :warning: 

https://user-images.githubusercontent.com/89014981/143271355-38b9362e-9708-42aa-ad02-28889f33bfc5.mp4

https://www.figma.com/file/uQYdD0E0ELDKi3skrduZpw/LLM-%2F-Notification-Center?node-id=3%3A9

### Type

Rebranding

### Context

[LL-8031]

### Parts of the app affected / Test plan

Notification center / Snackbar

[LL-8031]: https://ledgerhq.atlassian.net/browse/LL-8031?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -43,7 +43,8 @@ export interface BaseButtonProps extends Omit<ButtonProps, "type"> {
   eventProperties?: Object;
   // for testing
   testID?: string;
-}
+  type?: string;
+};
```
Comment (elbywan) [inline src/components/Button.tsx:47]: linter 🔴

File: src/components/RootNavigator/NotificationCenterNavigator.tsx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+              canPreventDefault: true,
+            });
+
+            if (!isActive && !event.defaultPrevented) {
+              navigation.navigate(route.name);
+            }
+          };
+
+          return (
+            <NavigationToggle
```
Comment (elbywan) [inline src/components/RootNavigator/NotificationCenterNavigator.tsx:46]: Can you double check if the `key` prop is needed here?

**Review by elbywan — APPROVED** [review]
LGTM 👌

File: src/components/RootNavigator/NotificationCenterNavigator.tsx (hunk 2 above)
Comment (nparigi-ledger) [reply src/components/RootNavigator/NotificationCenterNavigator.tsx]: Nice find, didn't had warning but it can't hurt, added

**Pull request merged**